test/utils/Spy.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const { assert } = require('chai');
+
+/**
+` * This class includes the functions to assert sinon spy on a function.
```

Comment:
```suggestion
 * This class includes the functions to assert sinon spy on a function.
```
Still not a good doc comment...